unfoldingWord/translationCore
# Disable select buttons for current project and tool

#### This pull request addresses:

Issue #1208, disabling of currently selected tool and project "Select" buttons



#### How to test this pull request:

Open a project and select a tool, then go back to the main modal and see that the Select button for that project and tool is disabled.


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword-dev/translationcore/1664)
